### ISSUE TITLE
Tool chain problem

### ISSUE THREAD

**Issue by viosoft** [mips-open-ide/Releases#8]
I have received the new MIPS Open IDE installation package 

I was able to successfully install it without a problem on my system. My system is a 64-bit Windows 10 Pro.

After downloading the toolchain from the location indicated in the MissingToolchain.png, and untarring it on my system at the location indicated in the BIN directory location.png,  and after I have modified the PATH variable as shown in PATH environment variable.png I continue to see the same assertion illustrated in MissingToolchain.png.

One other observation I have, is that the program (MIPS Open IDE) does not figure in the list of programs which may be uninstalled (see MissingMIPSOpenIDE.png).

Am I missing something?

Many thanks in advance,

-Dinu

**Comment by tuanha-viosoft** [comment]
@viosoft can you add the missing pictures (MissingToolchain.png, location.png, variable.png, MissingMIPSOpenIDE.png)?

**Comment by viosoft** [comment]
![PATH environment variable](https://user-images.githubusercontent.com/12651058/55050618-d1bb7b80-500e-11e9-9622-7c544f68379b.png)
![BIN directory location](https://user-images.githubusercontent.com/12651058/55050619-d1bb7b80-500e-11e9-975d-620cc48f530e.png)
![MIssingToolchain](https://user-images.githubusercontent.com/12651058/55050620-d1bb7b80-500e-11e9-96aa-da270a958a6d.png)
![MIssingMIPSOpenIDE](https://user-images.githubusercontent.com/12651058/55050621-d1bb7b80-500e-11e9-87a7-50300a50b9ce.png)

**Comment by tuanha-viosoft** [comment]
The MIPS Open IDE supports MTI only now. Please install MTI toolchains (mips-mti-elf, mips-mti-linux-gnu) and try again.

**Comment by dradian** [comment]
I have downloaded the MTI toolchain untarred to a different location (see **MissingMTIToolchain.png**).
![MIssingMTIToolchain](https://user-images.githubusercontent.com/643539/55103126-1f72cb00-5085-11e9-84e0-2ab3691f2437.png)
Then I have modified the Path as shown in the **New PATH environment variable.png**
![New PATH environment variable](https://user-images.githubusercontent.com/643539/55103184-3b766c80-5085-11e9-9f6e-8a2c6eba82e5.png)
I restarted the system, launch MIPS Open IDE from the desktop icon and observed the same **MissingToolchain.png** message as before. When pressing the close button, the MIPS Open IDE app closes as well.

IMPORTANT NOTE: If only MTI Toolchain builds are supported by the tool, then users may become confused as to which toolchain to download. Suggestion: either indicate this in the **Missing Toolchains.png** message box, or in the Release Notes for MIPS Open IDE, or remove the choices which are not compatible with MIPS Open IDE form the URL link which is provided in the **Missing Toolchains.png** message.

REMINDER NOTE: As mentioned before, having an uninstaller for the application is standard.
